src/main/java/com/spring/beans/Bicycle.java

```diff
@@ -0,0 +1,48 @@
+package com.spring.beans;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Setter
+@Getter
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON)
+@PropertySource({"classpath:spring/bicycle.properties"})
+public class Bicycle implements Transport {
+    private static Logger log = Logger.getLogger(Car.class.getName());
+
+    @Value("${speed}")
+    private int speed;
+
+    @Value("${countOfWheels}")
+    private int countOfWheels;
+
+    @Autowired
```

Comment:
for future read about DI with lombok 
https://www.baeldung.com/spring-injection-lombok

Reply:
Thank you!